### PR TITLE
Cascader / dropdown / i18n spanish version

### PR DESCRIPTION
**Pull request by Gonzalo2310**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by Leopoldthecoder** [comment]
Could you rebase `es-doc` branch? Because some of the changes are already merged.

**Comment by Gonzalo2310** [comment]
I'm not sure I did it right.
It's my first collaboration.
Sorry for the inconvenience
 :)

**Comment by Gonzalo2310** [comment]
Everything in order now?

**Comment by Leopoldthecoder** [comment]
Yes, now the commits are correct. I'm leaving this PR open so that you can keep pushing commits here.

**Comment by Gonzalo2310** [comment]
Sorry because of the confusing commit.
It should have been "input-number. md spanish version".

**Comment by Leopoldthecoder** [comment]
No problem.

**Comment by Leopoldthecoder** [comment]
Hi, actually you don't have merge remote `es-doc` in this PR. Could you take these steps?

```bash
git reset 83f0536 --hard
```

```bash
git push origin es-doc -f
```

**Comment by Gonzalo2310** [comment]
@Leopoldthecoder I don't know if it's okay. Sorry for the difficulties

**Comment by Leopoldthecoder** [comment]
No problem.

File: examples/docs/es/select.md [hunk 1]
```diff
@@ -317,11 +319,12 @@ You can clear Select using a clear icon.
 ```
 :::
 
-### Basic multiple select
+### Selección múltiple básica
+
+Selección multiple utiliza tags para mostrar las opciones seleccionadas.
 
-Multiple select uses tags to display selected options.
+:::demo Configure el atributo `multiple` para `el-select` para habilitar el modo múltiple. En este caso, el valor del `v-model` será un array de opciones seleccionadas. By default the selected options will be displayed as Tags. You can collapse them to a text by using `collapse-tags` attribute.
```
Comment (Leopoldthecoder) [inline examples/docs/es/select.md]: Sorry I forgot something:
> By default the selected options will be displayed as Tags. You can collapse them to a text by using `collapse-tags` attribute.

This last sentence is added a couple of days ago, so could you send another PR for `collapse-tags`? Sorry for the trouble.

File: examples/docs/es/select.md [hunk 2]
```diff
@@ -654,58 +660,60 @@ Create and select new items that are not included in select options
...
-| disabled | whether Select is disabled | boolean | — | false |
-| value-key | unique identity key name for value, required when value is an object | string | — | value |
-| size | size of Input | string | large/small/mini | — |
-| clearable | whether single select can be cleared | boolean | — | false |
+### Select atributos
+| Atributo             | Descripción                              | Tipo     | Valores aceptados | Por defecto      |
+| -------------------- | ---------------------------------------- | -------- | ----------------- | ---------------- |
+| multiple             | si multiple-select esta activo           | boolean  | —                 | false            |
+| disabled             | si Select esta deshabilitado             | boolean  | —                 | false            |
 | collapse-tags | whether to collapse tags to a text when multiple selecting | boolean | — | false |
```
Comment (Leopoldthecoder) [inline examples/docs/es/select.md]: Also this line, please.

**Comment by Gonzalo2310** [comment]
Yesterday was a complicated day. I'm sorry I didn't answer. I'll make those corrections right away. Thanks to you

**Comment by Leopoldthecoder** [comment]
Don't be sorry. We are thankful for your efforts.

**Comment by Gonzalo2310** [comment]
Done.
